src/components/landing-page/landing-page.controller.ts

```diff
@@ -2,10 +2,39 @@ import * as angular from 'angular';
 
 
 export class LandingPageController implements angular.IController {
+  static $inject = ['$scope', '$timeout'];
 
   public ctrl: any = this;
+  private $scope: any;
+  private $timeout: any;
+
+  constructor ($scope: any, $timeout: any) {
+    this.$scope = $scope;
+    this.$timeout = $timeout;
+  }
 
   public $onInit() {
     this.ctrl.searchText = '';
   }
+
+  public $postLink() {
+    this.$scope.$watch(() => {
+      return this.ctrl.alerts;
+    }, () => {
+      _.forEach(this.ctrl.alerts, (alert: any) => {
```

Comment:
I don't like that this timer logic would have to be handled by every controller that wants to have toast notifications